cmd/lakectl/cmd/repo_create.go

```diff
@@ -52,7 +52,7 @@ var repoCreateCmd = &cobra.Command{
 //nolint:gochecknoinits
 func init() {
 	repoCreateCmd.Flags().StringP("default-branch", "d", DefaultBranch, "the default branch of this repository")
-	repoCreateCmd.Flags().String("storage-id", "", "the storage of this repository")
+	repoCreateCmd.Flags().String("storage-id", "", "")
```

Comment:
Can you please define the storage-id flag globally (see `withNoProgress` as example) and set its property to hidden so it won't show up in the CLI docs (`MarkHidden`)